index.html

```diff
@@ -1,19 +1,42 @@
 <!DOCTYPE html>
-<html lang="ru">
-    <head>
-        <meta charset="UTF-8">
-        <meta name="viewport" content="widh=device-widh, inicial-scale=1.0">
-        <title>ZEN</title>
-    </head>
-    <body>
-        <header>Шапка</header>
-        <p>
-        <section>СЕКЦИЯ 1</section>
-        <section>СЕКЦИЯ 2</section>
-        <section>СЕКЦИЯ 3</section>
-        <section>СЕКЦИЯ 4</section>
-        <section>СЕКЦИЯ 5</section>
-        </p>
-        <footer>Подвал</footer>
-    </body>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="widh=device-widh, inicial-scale=1.0">
+    <title>ZEN</title>
+  </head>
+  <body>
+    <!-- Шапка сайта -->  
+    <header>Шапка</header>
+    <!-- Секция 1 -->
+    <section class="section" id="warranty">
```

Comment:
Use id with value "download"
Use BEM methodology
Use class name "download" for block